types/react-form/index.d.ts

```diff
@@ -1,36 +1,25 @@
-// Type definitions for react-form 1.3
+// Type definitions for react-form 2.12
 // Project: https://github.com/tannerlinsley/react-form#readme
 // Definitions by: Cameron McAteer <https://github.com/cameron-mcateer>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.6
```

Comment:
Is 2.6 really needed for these typings to work? If you're not using any specific TypeScript features only available to 2.6, then please leave the version as is, to allow users of older TypeScript to benefit from the typings.

Reply:
It should be fine at 2.3, I just upped it to 2.6 because that's what I run on my machine,